### DISCUSSION TITLE
Easy go-gardian unblock

### DISCUSSION THREAD

**Discussion by The-Man-Is-Me** [3kh0/ext-remover#195] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
here is an easy go guardian unblock
javascript:(function(){let url = prompt("about:blank spoofer | what website do you want to spoof");win = window.open();win.document.body.style.margin = '0';win.document.body.style.height = '100vh';var iframe = win.document.createElement('iframe');iframe.style.border = 'none';iframe.style.width = '100%';iframe.style.height = '100%';iframe.style.margin = '0';iframe.src = url;win.document.body.appendChild(iframe);})();

3kh0 has much better site  https://3kh0.github.io/dump/about-blank-embed.html

**Comment by 3kh0** [comment]
@The-Man-Is-Me try this tool: https://3kh0.github.io/dump/about-blank-embed.html it makes it easier. Make sure you embed the page in a about:blank page

**Reply by Terrinn** [reply]
Can you put an example for if we wanted to open youtube for example

**Comment by ghost** [comment]
GoGuardian*, and you can use the one that I have that lets you use the actual site instead of an iframe

**Reply by Meta0Music** [reply]
this doesn't work for everything that isnt goguardian
my district uses iboss lol

**Reply by ghost** [reply]
@Meta0Music So, does iboss use an overlay? Anything that redirects should work

**Reply by synapt0** [reply]
@yeeteeyt It doesn't work for securly, even though it redirects.

**Reply by ghost** [reply]
@NotJantix could you do a screen record and show me if you don't mind?

**Reply by synapt0** [reply]
@yeeteeyt aigh bet just wait till school ends

**Comment by Sxri** [comment]
Nice method, our school uses something like that and it works for us, but then it gets blocked with an overlay of Fortinet because of the school wifi, we need our phones or a VPN (which we cant install on our chromebooks sadly) to get around the wifi patch.
